src/Bicep.Core.Samples/Files/InvalidExpressions_LF/main.bicep

```diff
@@ -246,4 +246,11 @@ var partialObject = {
   a # 22
   c :
   d  : %
-}
\ No newline at end of file
+}
+
+// dangling decorators
```

Comment:
>decorators [](start = 12, length = 10)

Might be worth adding a comment warning people not to add declarations after this point.

Reply:
Good suggestion. Added a note.